<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit_number">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_capitalletter">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_character">
      <property name="text">
       <string>Character</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_number">
      <property name="text">
       <string>Number</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QPushButton" name="generate">
      <property name="text">
       <string>Generate</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_passwordlength">
      <property name="text">
       <string>Password Length</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_capitalletter">
      <property name="text">
       <string>Capital Letter</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QTextBrowser" name="displaypassword"/>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEdit_passwordlength">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEdit_character">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QPushButton" name="clear">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_passwordlength</tabstop>
  <tabstop>lineEdit_capitalletter</tabstop>
  <tabstop>lineEdit_number</tabstop>
  <tabstop>lineEdit_character</tabstop>
  <tabstop>displaypassword</tabstop>
  <tabstop>generate</tabstop>
  <tabstop>clear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
